pkg/virt-handler/device-manager/deviceplugin/v1beta1/api.proto

```diff
@@ -47,11 +47,11 @@ message Empty {
 // DevicePlugin is the service advertised by Device Plugins
 service DevicePlugin {
 	// GetDevicePluginOptions returns options to be communicated with Device
-        // Manager
+    // Manager
```

Comment:
I do not think it is a good a idea to fix this here. Maybe a different PR is required.